src/app/pages/pages-routing.module.ts

```diff
@@ -7,66 +7,68 @@ import { Error404Component } from './errors/error-404/error-404.component';
 import { Error403Component } from './errors/error-403/error-403.component';
 import { WelcomeComponent } from './welcome/welcome.component';
 
-const routes: Routes = [{
-  path: '',
-  component: PagesComponent,
-  children: [
-    {
-      path: 'dashboard',
-      component: DashboardComponent,
-    },
-    {
-      path: 'welcome',
-      component: WelcomeComponent,
-    },
-    {
-      path: 'users',
-      loadChildren: () => import('./users/users.module')
-        .then(m => m.UsersModule),
-    },
-    {
-      path: 'surveys',
-      loadChildren: () => import('./surveys/surveys.module')
-        .then(m => m.SurveysModule),
-    },
-    {
-      path: 'families',
-      loadChildren: () => import('./families/families.module')
-        .then(m => m.familiesModule),
-    },
-    {
-      path: 'documents',
-      loadChildren: () => import('./documents/documents.module')
-        .then(m => m.DocumentsModule),
-    },
-    {
-      path: 'files',
-      loadChildren: () => import('./file-manager/file-manager.module')
-        .then(m => m.FileManagerModule),
-    },
-    {
-      path: 'notifications',
-      loadChildren: () => import('./notifications/notifications.module')
-        .then(m => m.NotificationsModule),
-    },
-    {
-      path: 'not-allowed',
-      component: Error403Component,
-    },
-    {
-      path: 'not-found',
-      component: Error404Component,
-    }, /*,
+const routes: Routes = [
+  {
+    path: '',
+    component: PagesComponent,
+    children: [
+      {
+        path: 'dashboard',
+        component: DashboardComponent
+      },
+      {
+        path: 'welcome',
+        component: WelcomeComponent
+      },
+      {
+        path: 'users',
+        loadChildren: () => import('./users/users.module')
+          .then(m => m.UsersModule)
+      },
+      {
+        path: 'surveys',
+        loadChildren: () => import('./surveys/surveys.module')
+          .then(m => m.SurveysModule)
+      },
+      {
+        path: 'families',
+        loadChildren: () => import('./families/families.module')
+          .then(m => m.familiesModule)
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/HEPIA-Association/association-WebClient/pullRequest?prid=7914646)
```suggestion
          .then((m) => m.familiesModule)
```